src/main/java/edu/bu/cs673/stockportfolio/domain/investment/quote/Quote.java

```diff
@@ -58,13 +64,22 @@ public Quote() {
      * @param accountLines A collection of all the different owned lots within an account.
      */
     public Quote(String companyName, String symbol, BigDecimal latestPrice,
-                 Long marketCap, List<AccountLine> accountLines) {
+                 Long marketCap, List<AccountLine> accountLines, String sector, Company company) {
```

Comment:
How come you're adding sector into the parameter list but never initialize it inside the constructor?

Reply:
You're right. Sector shouldn't be here.

Reply:
Fixed in upcoming commit.